client/src/client_proxy.rs

```diff
@@ -4,7 +4,7 @@
 use crate::{commands::*, grpc_client::GRPCClient, AccountData, AccountStatus};
 use admission_control_proto::proto::admission_control::SubmitTransactionRequest;
 use config::trusted_peers::TrustedPeersConfig;
-use crypto::signing::KeyPair;
+use crypto::signing::{KeyPair, PublicKey, Signature};
```

Comment:
Thanks for the PR. Note that the legacy `crypto::signing` is going to be replaced by  `nextgen_crypto`, as shown in #493 
I'm not reviewing the scope of your PR at this stage, but in any case make sure you rebase and update all invocation to the `nextgen` crate after #493 is merged. `crypto::signing` will be then deprecated and we discourage its use.

Reply:
Will do.

Reply:
So is it a blocker for the PR, or can we merge and then change after #493 is merged?

Reply:
Hi @oleiba , just merged #493, could you please rebase, squash and push? I need to pull locally and evaluate properly with some tests in mind.